src/main/resources/openapi.json

```diff
@@ -0,0 +1,1075 @@
+{
+  "openapi": "3.0.1",
```

Comment:
isn't this something that is autogenerated for swagger?

Reply:
It is. But first, we're making sure that spec is found at the right path. And second, we are exposing only Insights related endpoints to the plugin, not all of them